Tasks/DotNetCoreCLIV2/Tests/L0.ts

```diff
@@ -205,18 +205,6 @@ describe('DotNetCoreExe Suite', function () {
         done();
     });
 
-    it('build passes when zero match found with empty string', (done: MochaDone) => {
```

Comment:
why is this removed? and when will you add it back?

Reply:
https://github.com/microsoft/azure-pipelines-tasks/pull/12358#discussion_r386461029
The tests run with Mocha, so I will find out whether they can accept variable paths. I will add a user story to track.

Reply:
The way to make this work is to have a function (getLoggerPath()) which gets logger path and then mock that function to return a static path ("/dummy/logger/path") in test. this way the build command remain static and easy to validate.